.github/workflows/release-go.yml

```diff
@@ -0,0 +1,40 @@
+name: Build and Release
+
+# on:
+#   schedule:
+#     - cron:  '0 6 * * 1'
+
+# fixme just testing
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.13.x'
+      - run: |
+          export GIT_COMMIT=$(git rev-parse HEAD) && \
+          export GIT_REPOSITORY=$(git config --get remote.origin.url) && \
+          CGO_ENABLED=0 go build -ldflags "-X github.com/SAP/jenkins-library/cmd.GitCommit=${GIT_COMMIT} \
+                  -X github.com/SAP/jenkins-library/pkg/log.LibraryRepository=${GIT_REPOSITORY} \
+                  -X github.com/SAP/jenkins-library/pkg/telemetry.LibraryRepository=${GIT_REPOSITORY}" \
+            -o piper .
+        shell: bash
+      - run: ./piper version
+        shell: bash
+      - uses: actions/upload-artifact@v1
+        with:
+          name: piper
+          path: piper
+      - name: Calculate Release Version
+        run: |
+          set -x
+          npm i semver
```

Comment:
Could we install the tool globally, that way it should be already on the path, right?